manager/manager.go

```diff
@@ -757,23 +757,34 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 		if strings.HasPrefix(k, "prometheus") || strings.HasPrefix(k, "Prometheus") {
```

Comment:
nit: Can you change this code to be `strings.HasPrefix(strings.ToLower(k), "prometheus")` here and apply the same logic below to `jolokia` as well?